com.sleepfiles.OSCAR.yaml

```
@@ -55,7 +55,8 @@ modules:
       # the generated makefiles do not support make install(?)
       - install -D -m 755 oscar/OSCAR $FLATPAK_DEST/bin/OSCAR
       - install -D -m 644 Building/Linux/OSCAR.svg $FLATPAK_DEST/share/icons/hicolor/scalable/apps/com.sleepfiles.OSCAR.svg
-      - install -D -m 644 Building/Linux/OSCAR.png $FLATPAK_DEST/share/icons/hicolor/128x128/apps/com.sleepfiles.OSCAR.png
+      - convert Building/Linux/OSCAR.svg -resize 128x128 OSCAR-128.png
+      - install -D -m 644 OSCAR-128.png $FLATPAK_DEST/share/icons/hicolor/128x128/apps/com.sleepfiles.OSCAR.png
```

```suggestion
      - install -D -m 644 Building/Linux/OSCAR.png $FLATPAK_DEST/share/icons/hicolor/48x48/apps/com.sleepfiles.OSCAR.png
```

Just not lying on the size and leaving the SVG should work. If not, just the SVG alone is enough.

---

oh wait... sorry, commited too fast. I removed everything else and just left the SVG now.